src/test/java/lotto/MoneyTest.java

```diff
@@ -0,0 +1,31 @@
+package lotto;
+
+import lotto.model.Money;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MoneyTest {
+    @DisplayName("지불한 돈 출력")
+    @Test
+    void 돈_출력() {
+        assertThat(new Money(5000).getMoney()).isEqualTo(5000);
```

Comment:
```suggestion
        assertThat(new Money(5000).value()).isEqualTo(5000);
```

원시값 포장이니까 저는 `값 줘`라고 표현해도 좋지 않을까? 생각이되네요 ㅎ